homeassistant/components/tuya/const.py

```diff
@@ -497,7 +497,7 @@ class UnitOfMeasurement:
     ),
     UnitOfMeasurement(
         unit=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        aliases={"ug/m3", "µg/m3", "ug/m³"},
+        aliases={"ug/m3", "μg/m3", "\u00b5g/m3", "ug/m³"},
```

Comment:
This is not easy to read.
If the pylint plugin doesn't catch using the wrong coding of µ here, I'd suggest to use unicode literals in both cases, with a comment explaining.

Reply:
The pylint plugin does not test on arguments, added a comment and now only unichar is used